### ISSUE TITLE
Feature Request: Add Tab Width Customization in Alacritty

### ISSUE THREAD

**Issue by o3wiz** [alacritty/alacritty#8317]
I would like to request a feature to allow customization of tab width directly in Alacritty.
### Proposed Feature
Introduce a configuration option (e.g., tab_width) in the alacritty.toml file that allows users to specify the number of spaces a tab character should occupy.

For example:
```toml
[terminal]
tab_width = 4
```

**Comment by kchibisov** [comment]
No. If you know what you're doing, then you can compile it yourself with the tab width you want and all the relevant places updates as well.

**Comment by R3EBL** [comment]
wydm

> No. If you know what you're doing, then you can compile it yourself with the tab width you want and all the relevant places updates as well.

...
i heard that people solved problems here, not boast superiority and  disrespect other

**Comment by chrisduerr** [comment]
> i heard that people solved problems here, not boast superiority and disrespect other

Where did you hear that? Nobody ever claimed so.
